src/logic/hooks/useEstimateSafeCreationGas.tsx

```diff
@@ -32,7 +33,7 @@ const estimateGas = async (
   const [gasEstimation, gasPrice, feesPerGas] = await Promise.all([
     estimateGasForDeployingSafe(addresses, numOwners, userAccount, safeCreationSalt),
     calculateGasPrice(),
-    getFeesPerGas(),
+    isMaxFeeParam() ? getFeesPerGas() : { maxPriorityFeePerGas: 0, maxFeePerGas: 0 },
```

Comment:
Why do we fallback here instead of `getFeesPerGas()` ?

Reply:
Not all chains support it.

Reply:
Just challenging if that logic does not belong to `getFeesPerGas()` responsibility

Reply:
Good idea, I agree that they belong together. Lets refactor it in this PR!

Reply:
Pushed the refactor

Reply:
Getting an error after moving `isMaxFeeParam` call into `getFeesPerGas`
![Screenshot 2022-02-28 at 15 17 10](https://user-images.githubusercontent.com/5880855/155998315-c2c5aca6-9ec5-4f13-996f-bd88cc2aa77c.png)